src/App.js

```diff
@@ -10,10 +10,30 @@ import Contact from './components/Contact';
 import UserProfile from './components/UserProfile';
 import SignUp from './components/SignUp';
 import About from './components/About'
+import Animal from './components/Animal'
 
 function App() {
     const [cookies, setCookie, removeCookie] = useCookies(['user'])
 
+    // Warm up backend server to decrease latency
+    useEffect(() => {
+        const warmServer = async () => {
+            const res = await fetch(
+                `https://bring-me-home-backend.herokuapp.com/`,
+                {
+                    method: 'GET',
+                    headers: {'Content-type': 'application/json'}
+                }
+            )
+    
+            if (res.ok) {
+                console.log('hello world')
+            }
+        }
+
+        warmServer()
+    })
```

Comment:
If we only want the useEffect to run once we should put an empty array at the end 
useEffect(()=>{},[]) 
That way it only runs the function once and won't rerun it every time there is a state change in the App